elements/pl-number-input/pl-number-input.mustache

```diff
@@ -15,16 +15,30 @@
             <span class="input-group-text">{{label}}</span>
         </span>
         {{/label}}
-        <input name="{{name}}" type="text" class="form-control" size="35" {{^editable}}disabled{{/editable}} {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}} aria-describedby="basic-addon1" placeholder="{{shortinfo}}" />
-        <span class="input-group-append">
-            {{#suffix}}<span class="input-group-text">{{suffix}}</span>{{/suffix}}
-            <a class="btn btn-light border" role="button" data-toggle="popover" data-html="true" title="Number" data-content="{{info}}" data-placement="auto" data-trigger="focus" tabindex="0">
-                <i class="fa fa-question-circle" aria-hidden="true"></i>
-                {{#correct}}&nbsp;<span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>{{/correct}}
-                {{#partial}}&nbsp;<span class="badge badge-warning"><i class="fa fa-circle-o" aria-hidden="true"></i> {{partial}}%</span>{{/partial}}
-                {{#incorrect}}&nbsp;<span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i> 0%</span>{{/incorrect}}
-            </a>
-        </span>
+        <input name="{{name}}" type="text" class="form-control {{^display_append_span}}rounded-right{{/display_append_span}}" size="{{size}}" {{^editable}}disabled{{/editable}} {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}} aria-describedby="basic-addon1" placeholder="{{shortinfo}}" />
+          
+          <span class="input-group-append">
+            {{#suffix}}<span class="input-group-text {{^questionmark}}rounded-right{{/questionmark}}">{{suffix}}</span>{{/suffix}}
+            {{#questionmark}}
+              <a class="btn btn-light border" role="button" data-toggle="popover" data-html="true" title="Number" data-content="{{info}}" data-placement="auto" data-trigger="focus" tabindex="0">
```

Comment:
If `show-correct-answer == True` and the element has `#correct` then can we add the display of `{{a_tru}}` to the end of the `{{info}}`? That is, make the popover contain both the current `{{info}}` and also the identical text that we display for correct answers in the submission panel, with the same logic? I think this would help to avoid confusion for students by putting the correct answer information in both places.

Reply:
Sure, but what do you suggest for when the popup is disabled?  If `show-correct-answer` is going to default to True then it may make sense to hide it as well.  Just wondering what people's thoughts on this were.

Reply:
One straightforward option would be to enable the popover if `show-correct-answer` is True, but don't put the `{{info}}` string in it in that case, but rather only the true answer (so it would work like the popover in the submission panel).

That is, have two conditions for showing the popover next to the input element. First, if `show-help-text` is True then the popover displayed and contains `{{info}}`. Second, if `show-correct-answer` is True and we should be displaying the correct answer then also display the popover, containing the true answer. If both conditions hold then display the popover with both the info text and the true answer.

Does this system make sense? Any corner cases I'm forgetting?